templates/wiki/page_in_use.html

```diff
@@ -0,0 +1,10 @@
+{% extends "wiki/base.html" %}
+{% block title %}Wiki Error{% endblock %}
+{% block og_title %}Wiki Error{% endblock %}
+{% block og_description %}{% endblock %}
+{% block content %}
+    <div class="uk-container uk-container-small">
+        <h1>The page you requested is currently being edited</h1>
```

Comment:
That header is probably _huge_, isn't it? Maybe use a h3?

Reply:
Could wrap the entire thing with one of those alert boxes as well

Reply:
Switched to h3 and wrapped in alert.